clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"regexp"
+	"strconv"
+	"sync"
+	"time"
+
+	goerrors "github.com/go-errors/errors"
```

Comment:
Why don't we use idiomatic way instead of a separate library?
https://blog.golang.org/go1.13-errors#TOC_3.3.

Reply:
Nice, I wasn't aware that the is an idiomatic way :)
+1 to using it instead of 3rd part library

Reply:
Done.